Runtime/Scripts/GltfAssetBase.cs

```diff
@@ -73,6 +73,11 @@ public class GltfAssetBase : MonoBehaviour
             return success;
         }
 
+        public bool Instantiate(Transform opt)
```

Comment:
What is the added value of this method in comparison to `public bool Instantiate(ICodeLogger logger = null)` from line 67?

It removed some security checks, so I'm inclined to reject this version.